src/messages/src_authority.rs

```diff
@@ -26,7 +26,7 @@ pub enum SrcAuthority {
     /// Authority of a single node.
```

Comment:
This is a Peer as we are the Node and the only Node.

Reply:
I think the word `Node` was used only mean to refer to `Single`

How about change this to `Single` ?

I am just concerned that change to `Peer` could cause the same confusion when thinking from the `sender`'s perspective?